lib/api/CDataFrameTrainBoostedTreeRunner.cc

```diff
@@ -436,6 +439,9 @@ void CDataFrameTrainBoostedTreeRunner::runImpl(core::CDataFrame& frame) {
     this->validate(frame, dependentVariableColumn);
 
     switch (m_Task) {
+    case E_Encode:
+        m_BoostedTree = m_BoostedTreeFactory->buildForEncode(frame, dependentVariableColumn);
+        break;
```

Comment:
We probably need separate code for memory estimation, but this is still outstanding for incremental training so not necessarily part of this change.